src/components/content/content.d.ts

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 
 export interface ContentProps {
   align?: 'left' | 'center' | 'right';
-  as?: 'primary' | 'secondary';
+  variant?: 'primary' | 'secondary';
```

Comment:
This needs to be a breaking change if we're renaming this prop - you can point them towards this codemod in the breaking change message: https://github.com/Sage/carbon-codemod/tree/master/transforms/rename-prop

Reply:
Although it's out of scope for this deprecating SCSS task so maybe just leave it as `as` for now

Reply:
I would leave it but AS is a keyword for StyledSystem and it means different things ;/ 
https://styled-components.com/docs/api#as-polymorphic-prop

Reply:
Ah I see, in that case the breaking change message will be fine 👍🏻